core/plugins/org.polarsys.capella.core.preferences/src/org/polarsys/capella/core/commands/preferences/service/PropertyStore.java

```diff
@@ -26,52 +26,36 @@
 import org.eclipse.jface.preference.IPreferenceStore;
 import org.eclipse.jface.preference.PreferenceStore;
 import org.eclipse.jface.util.IPropertyChangeListener;
-import org.osgi.service.prefs.BackingStoreException;
-
 import org.polarsys.capella.common.tools.report.config.registry.ReportManagerRegistry;
 import org.polarsys.capella.common.tools.report.util.IReportManagerDefaultComponents;
 import org.polarsys.capella.core.preferences.Activator;
 
 /**
+ * Its overall complicated but mainly:
+ * 
+ * Unlike the default PreferenceStore, it stores all values even default ones. Indeed, Configuration projects and
+ * Capella projects can be shared between users and all preferences shall be shared between users even default ones.
+ * 
+ * It stores each preferences in the ProjectScope and as Properties using IProject.getPersistentProperty, which is idk
```

Comment:
Yes, I agree that it's strange to save project preferences using both ProjectScope (saved in project .settings) and IProject.getPersistentProperty (saved in the workspace with the path to the project). But it was like this before, so I think we could keep the same code.